code/game/objects/items/weapons/circuitboards/other.dm

```diff
@@ -7,4 +7,11 @@
 /obj/item/weapon/circuitboard/aicore
 	name = T_BOARD("AI core")
 	origin_tech = list(TECH_DATA = 4, TECH_BIO = 2)
-	board_type = "other"
\ No newline at end of file
+	board_type = "other"
+
+/obj/item/weapon/circuitboard/chem_master
+	name = T_BOARD("ChemMaster 3000")
+	build_path = /obj/machinery/chem_master
+	board_type = new /datum/frame/frame_types/machine
+	origin_tech = list(TECH_DATA = 3, TECH_MAGNET = 2)
+	req_components = list()
```

Comment:
Honestly, I'd think that the ChemMaster would require at least tier 2 matter bins along with some other stuff. Maybe manipulators. Can't see a reason for capacitors or micro-lasers, so would just be the first two to balance it out.

Reply:
I'd suggest two matter bins and two manipulators.